src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -624,7 +625,12 @@ void computeParents(ExecutionSpace const &space, Edges const &edges,
         // Comparison of weights as ints is the same as their comparison as
         // floats as long as they are positive and are not NaNs or inf
         static_assert(sizeof(int) == sizeof(float));
-        keys(e) = (key << shift) + reinterpret_cast<int const &>(edge.weight);
+        keys(e) = (key << shift) +
+#if KOKKOS_VERSION >= 40099
```

Comment:
No that's too brittle that would break anyone on develop who didn't pull this morning.

Reply:
OK if you use 4.1